common/tools/eslint-plugin-azure-sdk/docs/rules/ts-naming-options.md

```diff
@@ -8,10 +8,13 @@ Requires client method options parameter types to be suffixed with `Options` and
 
 ```ts
 class ServiceClient {
+  constructor(options: ServiceClientOptions) {
+    /* code */
+  }
   createItem(options: CreateItemOptions): Item {
     /* code to return instance of Item */
   }
-  upsertItem(options: UpsertItemOptions): Item {
+  insertItem(options: InsertItemOptions): Item {
```

Comment:
curious why this one changed?

Reply:
eslint is not nerdy enough to understand that `upsert` is a proper english word.

Reply:
Possibly relevant: [our guidelines like `upsert`](https://azure.github.io/azure-sdk/general_design.html#general-client-standardize-verbs).

Reply:
I reverted it back.